.changeset/slow-dancers-care.md

```diff
@@ -0,0 +1,5 @@
+---
+"@omnigraph/openapi": patch
+---
+
+Some additional tests
```

Comment:
I think we don't need a changeset because it is just a test

Reply:
I'll wait with merging it, add some more tests here.
I'll remove/extend the changeset accordingly